sympy/simplify/simplify.py

```diff
@@ -544,12 +575,13 @@ def done(e):
         return shorter(rv, collect_abs(rv))
 
     expr = sympify(expr)
-    kwargs = dict(
+    _kwargs = dict(
         ratio=kwargs.get('ratio', ratio),
```

Comment:
This code here is odd. It's impossible for `ratio` and so on to be in `kwargs` since they are explicit keyword arguments. I think it should just be `dict(ratio=ratio, measure=measure, ...)`.